exercises/02-publish-subscribe/before/Divergent.Sales.API/WebApiConfig.cs

```diff
@@ -1,33 +1,37 @@
-﻿using Castle.Windsor;
+﻿using System.Net.Http.Formatting;
+using System.Web.Http;
+using Castle.Windsor;
 using Newtonsoft.Json.Serialization;
+using Owin;
 using Radical.Bootstrapper.Windsor.AspNet.Infrastructure;
-using System.Net.Http.Formatting;
-using System.Web.Http;
 
 namespace Divergent.Sales.API
 {
-    public static class WebApiConfig
+    public class WebApiConfig
     {
-        public static void Register(HttpConfiguration config, IWindsorContainer container)
+        public static void Configure(IAppBuilder app, IWindsorContainer container)
         {
-            config.Formatters.Clear();
-            config.Formatters.Add(new JsonMediaTypeFormatter());
+            var config = new HttpConfiguration();
+
+            config.MapHttpAttributeRoutes();
+            config.Routes.MapHttpRoute(
+                name: "DefaultApi",
+                routeTemplate: "api/{controller}/{id}",
+                defaults: new { id = RouteParameter.Optional }
+            );
+            config.EnableCors();
 
             config.DependencyResolver = new WindsorDependencyResolver(container);
 
+            config.Formatters.Clear();
+            config.Formatters.Add(new JsonMediaTypeFormatter());
+
             config.Formatters
                 .JsonFormatter
                 .SerializerSettings
                 .ContractResolver = new CamelCasePropertyNamesContractResolver();
 
-            // Web API routes
-            config.MapHttpAttributeRoutes();
-
-            config.Routes.MapHttpRoute(
-                name: "DefaultApi",
-                routeTemplate: "api/{controller}/{id}",
-                defaults: new { id = RouteParameter.Optional }
-            );
+            app.UseWebApi(config);
         }
     }
-}
\ No newline at end of file
+}
```

Comment:
It looks like quite a lot of statements have been re-ordered here. Would it be better to preserve the ordering to simplify the diff?